SUMMARY.md

```diff
@@ -47,6 +47,7 @@
 ## Documentation
 
 * [Architecture](docs/architecture.md)
+* [Upstream Service Integration](docs/data-flow.md)
```

Comment:
Because we all love consistency, could you make the filename reflect the document title? I'm also not sure "Upstream Service Integration" is as understandable as "Supporter Data Flow", but feel free to argue that one.

Reply:
6a5ec54ff4eb12791fba3edc32a93f4558fc73ba